src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -87,6 +87,11 @@ internal static class FeatureOnOffOptions
         public static readonly Option2<bool?> OfferRemoveUnusedReferences = new(
             nameof(FeatureOnOffOptions), nameof(OfferRemoveUnusedReferences), defaultValue: null,
             storageLocations: new RoamingProfileStorageLocation($"TextEditor.{nameof(OfferRemoveUnusedReferences)}"));
+
+        public static readonly Option2<bool> MoveToEndOfExpressionOnSemiColon = new(
+            nameof(FeatureOnOffOptions), nameof(MoveToEndOfExpressionOnSemiColon), defaultValue: true,
```

Comment:
i think the feature is called CompleteStatement, so MoveToEndOfExpression is inconssitent with our namign here.

Reply:
I didn't see this fixed?

Reply:
Pushed. You're up late 😛